app/section/sections/connectivity-and-resolution/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "connectivity-and-resolution",
+  "title": "Browser based - connectivity and resolution",
+  "mainAdvice": "Answer the following questions about your browser based Solution",
+  "additionalAdvice": [
+    "This information will be visible to buyers on your Solution page.",
+    "* indicates required field."
+  ],
+  "questions": {
+    "minimum-connection-speed": {
+      "mainAdvice": "Select the minimum connection speed required for your Solution to function?*",
+      "additionalAdvice": "Please select below the minimum bandwidth requirement.",
+      "type": "combobox-options",
+      "options": {
+        "": "Please select",
+        "0.5Mbps": "0.5Mbps",
+        "1Mbps": "1Mbps",
+        "1.5Mbps": "1.5Mbps",
+        "2Mbps": "2Mbps",
+        "3Mbps": "3Mbps",
+        "5Mbps": "5Mbps",
+        "8Mbps": "8Mbps",
+        "10Mbps": "10Mbps",
+        "15Mbps": "15Mbps",
+        "20Mbps": "20Mbps",
+        "30Mbps": "30Mbps",
+        "Higher than 30Mbps": "Higher than 30Mbps"
+      },
+      "errorResponse": {
+        "required": "Please select the minimum connection speed required"
+      }
+    },
+    "minimum-desktop-resolution": {
+      "mainAdvice": "Select the minimum desktop aspect ratio and screen resolution?",
+      "additionalAdvice": "Please select below the minimum resolution required for your Solution to function on desktop.",
+      "type": "combobox-options",
+      "options": {
+        "": "Please select",
+        "16:9 - 640 x 360": "16:9 - 640 x 360",
+        "4:3 - 800 x 600": "4:3 - 800 x 600",
```

Comment:
is this really how the data comes back? 👀

Reply:
haha yes. Nice ain't it. 
This is what was is in the prototypes and I agree it isn't nice. Would be nice if we gave the option to select an aspect ratio first and based upon the section show the relevant resolution. But without javascipt on the client side not sure how we can achieve that. I think it's definitely something we can revisit and look to improve.